docs/hub/datasets-dask.md

```diff
@@ -0,0 +1,39 @@
+# Dask
+
+[Dask](https://github.com/dask/dask) is a parallel and distributed computing library that scales the existing Python and PyData ecosystem.
+Since it uses [fsspec](https://filesystem-spec.readthedocs.io) to read and write remote data, you can use the Hugging Face paths (`hf://`) to read and write data on the Hub:
+
+First login using 
+
+```
+huggingface-cli login
```

Comment:
maybe add a link, so that people who what to login programmatically, instead of using the CLI, have not to search too much in the docs

Reply:
(link to the login docs: https://huggingface.co/docs/huggingface_hub/quick-start#login)